cpp/dolfinx/fem/DirichletBC.h

```diff
@@ -448,7 +517,22 @@ class DirichletBC
     }
   }
 
-  /// Set markers[i] = true if dof i has a boundary condition applied.
+  /// Set markers[i] = 1 if cell i has a dof in with boundary condition
+  /// applied. Value of markers[i] is not changed otherwise.
+  /// @param[in,out] markers Entry markers[i] is set to true if cell i in
```

Comment:
What does `Entry` mean?

What are the preconditions on `markers`? Use the doxygen syntax for preconditions to document.